docs/source/cuda/cuda_array_interface.rst

```diff
@@ -104,3 +104,23 @@ Differences with CUDA Array Interface (Version 1)
 Versions 0 and 1 of the CUDA Array Interface neither clarified the
 **strides** attribute for C-contiguous arrays nor specified the treatment for
 zero-size arrays.
+
```

Comment:
Whilst we're fixing up this doc, this entire section is incorrectly indented:
https://github.com/numba/numba/pull/5104/files#diff-017e6b3745f25f98ede11dfc0ede29caR26-R73

The paragraph after the list item needs indenting to the same level as the first character of the list item, i.e. 2 spaces.

Any chance you could fix this please whilst sorting out the issue below? Thanks!